doc/cargo_command_execution.md

```diff
@@ -75,6 +75,17 @@ The possible values:
 * Some object (`{ "RUST_BACKTRACE": 1 }`)
 * `null`
 
+### Setting An Action To Handle Starting A New Command If There Is Another Running Command
```

Comment:
`### Setting An Action To Handle Starting A New Command If There Is Another Command Running`